scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -64,7 +64,16 @@ abstract class SchemedSource extends Source {
  */
 trait LocalSourceOverride extends SchemedSource {
   /** A path to use for the local tap. */
-  def localPath: String
+  def localPaths: Iterable[String]
+
+  // By default, we write to the last path for local paths
+  def localWritePath = localPaths.last
+
+  object CastFileTap {
```

Comment:
let's not put this as an inner object. Can we make it a `private[scalding]` object in FileSource.scala?